.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,113 @@
+See the [Scientific Python Developer Guide][spc-dev-intro] for a detailed
+description of best practices for developing scientific packages.
+
+[spc-dev-intro]: https://scientific-python-cookie.readthedocs.io/guide/intro
```

Comment:
This link appears to be broken @lsetiawan

Reply:
@uwcdc could you see what's going on here? Thanks!

Reply:
@ConnorStoneAstro the Scientific Python link changed with the development version and needed to be updated in the CONTRIBUTING.md file to the following stable version:

https://scientific-python-cookie.readthedocs.io/en/stable/